### PR TITLE
Enable node-e2e tests for 1.4 release.

### PR DESCRIPTION
**Pull request by dchen1107**
cc/ @pwittrock @Random-Liu

---
This change is [Reviewable](https://reviewable.io/reviews/kubernetes/test-infra/571)

### CODE REVIEW

File: jenkins/job-configs/kubernetes-jenkins/node-e2e.yaml [hunk 1]
```diff
@@ -89,7 +89,7 @@
             browser: githubweb
             browser-url: 'https://github.com/{repoName}'
             branches:
-                - 'master'
+                - '{branch}'
```
Comment (Random-Liu) [inline jenkins/job-configs/kubernetes-jenkins/node-e2e.yaml]: You can set default to `master`.
Default value can be set like [this](https://github.com/kubernetes/test-infra/blob/master/jenkins/job-configs/kubernetes-jenkins/continuous-docker-validation.yaml#L70-L72)

File: jenkins/job-configs/kubernetes-jenkins/node-e2e.yaml [hunk 2]
```diff
@@ -140,19 +140,36 @@
...
+            branch: 'master'
             gitbasedir: 'k8s.io/kubernetes'
             owner: 'pwittroc@google.com'
             shell: 'test/e2e_node/jenkins/e2e-node-jenkins.sh test/e2e_node/jenkins/jenkins-ci.properties'
     jobs:
         - '{gitproject}-gce-e2e-ci'
+
+- project:
+    name: node-gce-e2e-1.4
+    repoName: 'google/cadvisor'
```
Comment (Random-Liu) [inline jenkins/job-configs/kubernetes-jenkins/node-e2e.yaml]: nit: IIUC, we don't need this line anymore, maybe we can also remove the one above.

**Comment by dchen1107** [comment]
Addressed your comments, ptal? Thanks!

**Comment by Random-Liu** [comment]
LGTM.

**Pull request merged**